editor/src/clj/editor/library.clj

```diff
@@ -108,13 +108,21 @@
 
 (defn- make-basic-auth-headers
   [^String user-info]
-  {"Authorization" (format "Basic %s" (str->b64 user-info))})
+  (let [user-info-parts (str/split user-info #":")
+        username (get user-info-parts 0)
+        password (get user-info-parts 1)]
```

Comment:
I think you want to provide a default `""` in case there is no `:` in the user-info string. This is similar to what you do in `Project.java`
```suggestion
        password (get user-info-parts 1 "")]
```

Reply:
Good point. Added fallback to empty string.